packages/docs-examples/components/toggle/toggle-error/toggle-error-example.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, signal } from '@angular/core';
+import { ThemePalette } from '@koobiq/components/core';
+import { KbqToggleModule } from '@koobiq/components/toggle';
+
+/**
+ * @title Toggle Error
+ */
+@Component({
+    standalone: true,
+    selector: 'toggle-error-example',
+    imports: [KbqToggleModule],
+    host: {
+        class: 'layout-margin-5xl layout-align-center-center layout-row'
+    },
+    template: `
+        <kbq-toggle [checked]="value()" [color]="color" (change)="value.set($event.checked)">Vibration</kbq-toggle>
+    `
+})
+export class ToggleErrorExample {
+    value = signal(false);
```

Comment:
это можно не исправлять, на будущее

по возможности лучше использовать [`readonly` модификатор](https://github.com/angular-eslint/angular-eslint/blob/af5386760858602db8a70f70ee06fc5991850c6f/packages/eslint-plugin/docs/rules/prefer-signals.md) для сигналов, меньше придется исправлять при включении правила

```suggestion
    readonly value = signal(false);
```